app/src/main/java/com/infomaniak/mail/data/models/mailbox/Mailbox.kt

```diff
@@ -99,7 +101,7 @@ class Mailbox : RealmObject {
     inline val notificationGroupId get() = uuid.hashCode()
     inline val notificationGroupKey get() = uuid
 
-    inline val isValid get() = isPasswordValid && !isLocked
+    inline val isValid get() = isValidInLDAP && !isLocked && hasValidPassword
```

Comment:
We could rename to something this `isAccessible` like iOS to avoid any confusion with the api field